volumina/view3d/overview3d.py

```diff
@@ -62,6 +76,15 @@ def _adjust_axes(x, y, z):
         """
         return x, y, z
 
+    def _remove_3d_view(self, display_string: str):
```

Comment:
I wonder if, rather than allowing a normal GlView to be created, and then have it replaced here, if it wouldn't be better to hide the declaration of the real GLView in the cases where OpenGL is not supported.
```python
if gl_version_sufficient:
    class GlView:
        "Real GlView"
else:
    class GlView:
        "Fake GlView"
```

That way other consumers of GLView will also have their implementation mocked automatically, plus we might dodge some traps of not setting up the mock properly, or failing to tear down the real one the correct way

Reply:
Can we create both and use only one of them?

```python
class GLView_Real:
    ...
class GLView_Fake:
    ....
GLView = GLView_Real if OPENGL_VERSION >= OPENGL_VERSION_MIN else GLView_Fake
```

Reply:
thought about it at the beginning, too, but found the "big if" pattern very ugly (just aesthetically), but I see the benefits. I'll give it a go, since you guys not seem to be repelled :)

Reply:
Okay, I did not have success in checking GL version setting the context myself. Does anyone else want a go with it? Otherwise I'll merge this an open an issue.

Reply:
Maybe we can just set a special environment variable `VOLUMINA_ENABLE_STUB_GLVIEW=1` and check for it in the code? It seems like this is needed only for tests, and if somebody wants to actually use volumina with an old OpenGL version, they need to install drivers anyway.

Reply:
well, the other use case is x-forwarding which was reportedly being problematic.

Reply:
> Okay, I did not have success in checking GL version setting the context myself. Does anyone else want a go with it? Otherwise I'll merge this an open an issue.

Hell, I always wanted to suffer under OpenGL =D
If you open the issue I'll get to it as soon as I get some time to breathe. But it's still useful to get this merged anyway